code/src/GridAPI/Rows.ts

```diff
@@ -15,20 +15,44 @@ namespace GridAPI.Rows {
         gridID: string,
         rowNumber: number,
         className: string
-    ): void {
+    ): string {
         PerformanceAPI.SetMark('Rows.AddClass');
+        let returnMessage = {
+            isSuccess: true,
+            message: 'Success',
+            code: OSFramework.Enum.ErrorCodes.GRID_SUCCESS
+        };
 
-        const grid = GridManager.GetGridById(gridID);
-
-        if (grid !== undefined) {
-            grid.features.rows.addClass(rowNumber, className, true);
+        if (!OSFramework.Helper.IsGridReady(gridID)) {
+            returnMessage = {
+                isSuccess: false,
+                message: 'Grid not found',
+                code: OSFramework.Enum.ErrorCodes.CFG_GridNotFound
+            };
+            return JSON.stringify(returnMessage);
+        }
+        try {
+            GridManager.GetGridById(gridID).features.rows.addClass(
+                rowNumber,
+                className,
+                true
+            );
+        } catch (error) {
+            returnMessage = {
+                isSuccess: false,
+                message: 'Error',
+                code: OSFramework.Enum.ErrorCodes.API_FailedAddClass
+            };
```

Comment:
here you can access to the properties of the type, e.g. `returnMessage.isSuccess = false`.
Also please change the name of the object to something different from "returnMessage" :) 
Repeated with https://github.com/OutSystems/outsystems-datagrid-reactive/pull/188